packages/ui/src/components/DateInput/__stories__/index.stories.tsx

```diff
@@ -3,26 +3,17 @@ import { DateInput } from '..'
 
 export default {
   component: DateInput,
-  decorators: [
-    Stories => (
-      <div style={{ height: 360 }}>
-        <Stories />
-      </div>
-    ),
-  ],
   title: 'Components/Data Entry/DateInput',
 } as Meta
-
 export { Playground } from './Playground.stories'
 export { Size } from './Size.stories'
-export { Exclude } from './Exclude.stories'
-export { Range } from './Range.stories'
-export { Uncontrolled } from './Uncontrolled.stories'
-export { Controlled } from './Controlled.stories'
 export { Error } from './Error.stories'
 export { Disabled } from './Disabled.stories'
 export { Required } from './Required.stories'
```

Comment:
Same for this story

![Screenshot 2024-11-13 at 11 07 01](https://github.com/user-attachments/assets/ad9c97da-6cf5-47c4-a9e7-c0609072f690)

Maybe check them all to see if the dropdown fit when opening